source/Mlos.Notebooks/SmartCacheOptimization.ipynb

```diff
@@ -18,7 +18,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
```

Comment:
@amueller to comment further, but I think we want to the output for this one for the webpages that get published.
At least until we can fix automatically rendering them during website generation.

Reply:
OK, I'll re-render these.

Reply:
We should have tests for this.

Reply:
Added an issue: #111.

Reply:
We also have #55 already